embeddingapi/embedding-build-android-tests/crosswalk-version

```diff
@@ -0,0 +1 @@
+14.43.343.20
```

Comment:
I suddenly find it will be better to use https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/VERSION instead of this crosswalk-version file.